src/test/resources/features/board.feature

```diff
@@ -1,6 +1,6 @@
 Feature: board is retrieved
 
-  Scenario: client makes call to GET /board
-    When the client calls /board
+  Scenario: client makes call to GET /boards
+    When the client calls /boards/1
```

Comment:
I can see that we will hard code this request in the front end for our single board for the moment.